src/Libraries/CoreNodes/Math.cs

```diff
@@ -2,9 +2,11 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
+using System.Windows.Media;
```

Comment:
we can't do this. These libraries need to be cross platform.

Reply:
what is using this anyway?

Reply:
Not sure. Removed.